src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -270,6 +279,51 @@ export default {
 		};
 	},
 	computed: {
+		loanCallouts() {
```

Comment:
Please create a follow-up ticket that turns this into a util and creates unit tests against it. We'll need to unit test all of the cases we expect, such the ones in the comments mentioned on this PR.

Reply:
This was turned into a util.
Here is the ticket for unit testing https://kiva.atlassian.net/browse/CORE-1080